bip-ctv.mediawiki

```diff
@@ -358,7 +358,7 @@ designers.
 
 However, committing to the index eliminates key-reuse vulnerability to the half-spend problem. As
 potentially reused CHECKTEMPLATEVERIFY scripts commit to being spent at the same index, they cannot
-be spent at the same index. This makes it safer to design wallet vault contracts without half-spend
+be spent at the different index. This makes it safer to design wallet vault contracts without half-spend
```

Comment:
Actually same is correct here.

if you want to make this one more clear it should read "they cannot both be spent at the same index, which implies that they cannot be spent in the same transaction."